### ISSUE TITLE
Unrecognized Arguments -n

### ISSUE THREAD

**Issue by stefanhuber1993** [kootenpv/whereami#28]
Hey there, 

I was trying to use 'whereami'. I use python 2.7, and installed whereami by pip.
I get the following error message.
```bash
>>> whereami learn -l work2 -n 100
usage: whereami [-h] [--version]
                {predict,predict_proba,crossval,locations,ls,learn,rename} ...
whereami: error: unrecognized arguments: -n 100
```
Best
Stefan

**Comment by stefanhuber1993** [comment]
#29 Pull Request

**Comment by kootenpv** [comment]
Solved by PR #29 indeed, thanks! Available as of `whereami==0.3.63`.

To upgrade in such a short time, make sure to avoid the cache:

    pip install -U --no-cache whereami

Let me know if it worked for you.

**Comment by stefanhuber1993** [comment]
Works nicely indeed, thanks.
